.nvmrc

```diff
@@ -1 +1 @@
-10
+12
```

Comment:
I think we can remove Node.js support for v10 (2018) from CI as well. Maintenance end of that version is in a little more than a month: 2021-04-30

Reply:
Let's do that in another PR.

Reply:
See, #648